### PR TITLE
Fix README to use hostfxr download links

### PR DESCRIPTION
**Pull request by schellap**
skip ci please

@brthor PTAL and confirm if this is the ordering you'd expect. Also, please let me know what the fourth column is for.

### CODE REVIEW

**Comment by schellap** [comment]
Fixes #3621

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ Ubuntu Installers
 |         |Version |Installers|
```
Comment (brthor) [inline README.md]: right above this 

> we have three separate component packages  

Change to 
> we have four separate component packages

**Comment by brthor** [comment]
Links look good, small comment on some of the wording, then :shipit:

File: README.md (hunk 1 above)
Comment (schellap) [inline README.md]: I changed it to "we have separate component packages"

**Pull request merged**